options.py

```diff
@@ -70,14 +70,14 @@ def __init__(self, library=1, configs=0, transform=0):
         self.load_file        = None        # load previous model weights when training if it exists
 
         # USE YOUR OWN DATA DIR: NEED ABSOLUTE PATH!
```

Comment:
can remove my comment here, I forgot I can use './' when writing this part of code XD